packages/react/turbo.json

```diff
@@ -2,11 +2,10 @@
   "extends": ["//"],
   "tasks": {
     "test": {
-      "dependsOn": ["^@clerk/localizations#build", "^build"],
```

Comment:
Let's keep `^build` as I don't believe need to build the current package in order to test it.

Reply:
Done